### PR TITLE
Created /xy command and complimentary test

### PR DESCRIPTION
**Pull request by BrianWisniewski**
- [x] You have read the [Odin Bot README/Contributing Guide](https://github.com/TheOdinProject/odin-bot-v2/blob/master/README.md).

#### 1.Describe the changes made:

I Created a /xy command and test to go with it under simpleCommands.js and simpleCommands.test.js. I already talked to @wise-king-sullyman in the discord under the bot playground text channel and he gave me the ok send my pr.

#### 2. If this PR modifies a command, please provide a screenshot of the passing tests below. NOTE: Your PR will not be merged if you have any failing test cases. 
image

#### 3. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#128

### CODE REVIEW

File: botCommands/simpleCommands.js [hunk 1]
```diff
@@ -110,6 +110,12 @@ const justDoIt = {
 };
 registerBotCommand(justDoIt.regex, justDoIt.cb);
 
+const xy = {
+  regex: /(?<!\S)\/xy(?!\S)/,
+  cb: () => 'What problem are you *really* trying to solve? Check out this article to help others better understand your question: https://xyproblem.info/',
```
Comment (twalton83) [inline botCommands/simpleCommands.js:115]: Small change, let’s make sure we remove the Discord embed around the link. See the question command and how we handle the link there.

**Review by twalton83 — CHANGES_REQUESTED** [review]
Just one small change, thank you!

File: botCommands/simpleCommands.js (hunk 1 above)
Comment (BrianWisniewski) [reply botCommands/simpleCommands.js:115]: What do you mean by "Discord Embed"?

Comment (BrianWisniewski) [reply botCommands/simpleCommands.js]: I looked over the question command under simple commands and other than the bold markdown notation, everything else seems to be the same. What exactly should I add?

Comment (twalton83) [reply botCommands/simpleCommands.js]: We want arrow brackets around the link: https://support.discord.com/hc/en-us/articles/206342858--How-do-I-disable-auto-embed-

**Comment by twalton83** [comment]
Your snapshot tests no longer match because you updated the command, go ahead and update the snapshot tests and push those changes, after that, I can merge this.

File: botCommands/simpleCommands.js (hunk 1 above)
Comment (BrianWisniewski) [reply botCommands/simpleCommands.js:115]: I made a commit with the arrow brackets but it failed the test. The question command under simpleCommands.js does not currently have brackets.

**Comment by BrianWisniewski** [comment]
Thank you! That makes sense. I'll get that done!

**Review by twalton83 — APPROVED** [review]
Appreciate this, thank you.

**Pull request merged**